rai/lib/numbers.hpp

```diff
@@ -35,6 +35,8 @@ union uint128_union
 	bool decode_hex (std::string const &);
 	void encode_dec (std::string &) const;
 	bool decode_dec (std::string const &);
+	std::string format_balance (rai::uint128_t scale, int precision, bool group_digits);
```

Comment:
Please make these methods `const` if it's possible.

Reply:
Done